package-lock.json

```diff
@@ -2,6 +2,7 @@
   "name": "vaadin-context-menu",
```

Comment:
Is it needed to change package-lock.json?

Reply:
Done